src/views/Rules/ListRuleRevisions/index.jsx

```diff
@@ -82,8 +89,44 @@ function ListRuleRevisions(props) {
     });
   };
 
-  // TODO: Add logic to restore a revision
-  const handleRestoreClick = () => {};
+  const handleRestoreClick = revision => () =>
+    setDialogState({
+      ...dialogState,
+      open: true,
+      body: `Rule ${
+        revision.alias ? revision.alias : revision.rule_id
+      } will be reverted to data version ${revision.data_version}.`,
+      title: 'Revert rule?',
+      confirmText: 'Revert',
```

Comment:
![Screen Shot 2019-08-16 at 10 17 00 AM](https://user-images.githubusercontent.com/3766511/63173966-ff7d7d00-c00e-11e9-8376-844e7c3a6bd3.png)

I think we should either change `Restore` to `Revert` in the table or have the dialog say `Restore Rule?` and have `Restore` as `confirmText` to make sure we are consistent.